guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -34,6 +34,12 @@ ifndef::satellite,orcharhino[]
 Specifying an operating system is required when you register machines without `subscription-manager`, such as Debian or Ubuntu.
 endif::[]
 . Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through.
+[NOTE]
+====
+If you have {SmartProxies} behind a load balancer, the host's content source is set to one of the {SmartProxies} behind the load balancer. 
+This {SmartProxy} might not be the same as the one you select here.
+====
```

Comment:
Feel free to ignore this suggestion if it makes no sense from a technical point of view.

"A {SmartProxy} behind a load balancer takes precedence over a {SmartProxy} selected in the {ProjectWebUI} as the host's content source."